<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaiaText2Ascii</class>
 <widget class="QMainWindow" name="MaiaText2Ascii">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaiaText2Ascii</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>58</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bin:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>33</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_hex">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>33</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>ASCII value</string>
    </property>
    <property name="maxLength">
     <number>1000000</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_bin">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>58</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>8 bits binary value</string>
    </property>
    <property name="maxLength">
     <number>1000000</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_text">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>8</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter a char</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>1000000</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>8</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Text:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>471</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_clear">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>8</y>
      <width>10</width>
      <height>70</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clear all (CTRL+L)</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label_bg</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>line_hex</zorder>
   <zorder>line_bin</zorder>
   <zorder>line_text</zorder>
   <zorder>label_7</zorder>
   <zorder>b_clear</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>line_text</tabstop>
  <tabstop>line_hex</tabstop>
  <tabstop>line_bin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
